Xamarin.Forms.Core/SwipeItems.cs

```diff
@@ -15,6 +16,18 @@ public SwipeItems()
 			_internal.CollectionChanged += OnSwipeItemsChanged;
 		}
 
+		public SwipeItems(IList<ISwipeItem> swipeItems)
```

Comment:
```suggestion
		public SwipeItems(IEnumerable<ISwipeItem> swipeItems)
```

ObservableCollection takes an IEnumerable<T> so it seems like this should just be the same

Reply:
Right. I have changed the constructor.